lib/posix/pthread_mutex.c

```diff
@@ -78,7 +78,7 @@ int pthread_mutex_timedlock(pthread_mutex_t *m,
 			    const struct timespec *abstime)
 {
 	s32_t timeout = (s32_t)timespec_to_timeoutms(abstime);
-	return acquire_mutex(m, timeout);
+	return acquire_mutex(m, K_MSEC(timeout));
```

Comment:
Use `SYS_TIMEOUT_MS()` instead which will convert `SYS_FOREVER_MS` to `K_FOREVER` automatically.

Reply:
Specifically here there's no either SYS_FOREVER_MS or K_FOREVER.

Reply:
I see, thanks